packages/11ty/_plugins/figures/annotation/README.md

```diff
@@ -0,0 +1,21 @@
+# Annotations
+
+## Annotation Set Model in `figures.yaml`
+| Property    | Description                   |
+| ----------- | ----------------------------  |
+| input       | `checkbox`|`radio` (default)  |
+| items       | Array\{Annotation Items\}     |
+| title       | The fieldset title for the UI |
+
+## Annotation Item Model in `figures.yaml`
+| Property | Description                                                  |
+| -------- | ------------------------------------------------------------ |
+| id       | Unique id (optional)
+| src      | The image file path relative to `iiifConfig.inputDir` |
+| label    | The input label; if not provided, the filename converted to title case will be used. |
```

Comment:
Adding an example of the yaml for each of these to this README may be helpful, thoughts?
```suggestion

Annotations and Annotation Sets are modeled in the `figures.yaml` data file

## `AnnotationSet`
| Property    | Description                   |
| ----------- | ----------------------------  |
| input       | `checkbox`|`radio` (default)  |
| items       | Array\{Annotation Items\}     |
| title       | `fieldset` title for the UI   |

## `Annotation` Item
| Property | Description                                                  |
| -------- | ------------------------------------------------------------ |
| id       | Unique id (optional)
| src      | The image file path relative to `iiifConfig.inputDir` |
| label    | The input label; if not provided, the filename converted to title case will be used. |
```